extensions/amp-3q-player/0.1/amp-3q-player.js

```diff
@@ -1,26 +1,27 @@
-import {Deferred} from '#core/data-structures/promise';
-import {removeElement} from '#core/dom';
+import { Deferred } from '#core/data-structures/promise';
+import { removeElement } from '#core/dom';
 import {
   fullscreenEnter,
   fullscreenExit,
   isFullscreenElement,
 } from '#core/dom/fullscreen';
-import {isLayoutSizeDefined} from '#core/dom/layout';
-import {PauseHelper} from '#core/dom/video/pause-helper';
+import { isLayoutSizeDefined } from '#core/dom/layout';
+import { setStyle } from '#core/dom/style';
+import { PauseHelper } from '#core/dom/video/pause-helper';
 
-import {Services} from '#service';
-import {installVideoManagerForDoc} from '#service/video-manager-impl';
+import { Services } from '#service';
+import { installVideoManagerForDoc } from '#service/video-manager-impl';
 
-import {getData, listen} from '#utils/event-helper';
-import {dev, userAssert} from '#utils/log';
+import { getData, listen } from '#utils/event-helper';
+import { dev, userAssert } from '#utils/log';
 
 import {
   createFrameFor,
   objOrParseJson,
   redispatch,
 } from '../../../src/iframe-video';
-import {addParamToUrl} from '../../../src/url';
-import {VideoEvents_Enum} from '../../../src/video-interface';
+import { addParamToUrl } from '../../../src/url';
+import { VideoEvents_Enum } from '../../../src/video-interface';
```

Comment:
Looks like the `prettier` config used here doesn't match the repo, so you're getting CI errors.